ignored-tests.json

```diff
@@ -0,0 +1,179 @@
+  "Chromium-Specific Launcher tests Puppeteer.launch |pipe| option should fire \"disconnected\" when closing with pipe",
+  "Chromium-Specific Launcher tests Puppeteer.launch |pipe| option should support the pipe argument",
+  "Chromium-Specific Launcher tests Puppeteer.launch |pipe| option should support the pipe option",
+  "Chromium-Specific Page Tests Page.setRequestInterception should work with intervention headers",
+  "Coverage specs CSSCoverage resetOnNavigation should NOT report scripts across navigations",
+  "Coverage specs CSSCoverage resetOnNavigation should report stylesheets across navigations",
+  "Coverage specs CSSCoverage should ignore injected stylesheets",
+  "Coverage specs CSSCoverage should report multiple stylesheets",
+  "Coverage specs CSSCoverage should report sourceURLs",
+  "Coverage specs CSSCoverage should report stylesheets that have no coverage",
+  "Coverage specs CSSCoverage should work with a recently loaded stylesheet",
+  "Coverage specs CSSCoverage should work with complicated usecases",
+  "Coverage specs CSSCoverage should work with empty stylesheets",
+  "Coverage specs CSSCoverage should work with media queries",
+  "Coverage specs CSSCoverage should work",
+  "Coverage specs JSCoverage includeRawScriptCoverage should include rawScriptCoverage field when enabled",
+  "Coverage specs JSCoverage includeRawScriptCoverage should not include rawScriptCoverage field when disabled",
+  "Coverage specs JSCoverage resetOnNavigation should NOT report scripts across navigations when enabled",
+  "Coverage specs JSCoverage resetOnNavigation should report scripts across navigations when disabled",
+  "Coverage specs JSCoverage should ignore eval() scripts by default",
+  "Coverage specs JSCoverage should ignore pptr internal scripts if reportAnonymousScripts is true",
+  "Coverage specs JSCoverage should not hang when there is a debugger statement",
+  "Coverage specs JSCoverage should not hang when there is a debugger statement",
+  "Coverage specs JSCoverage should not ignore eval() scripts if reportAnonymousScripts is true",
+  "Coverage specs JSCoverage should report multiple scripts",
+  "Coverage specs JSCoverage should report right ranges for \"per function\" scope",
+  "Coverage specs JSCoverage should report right ranges",
+  "Coverage specs JSCoverage should report scripts that have no coverage",
+  "Coverage specs JSCoverage should report sourceURLs",
+  "Coverage specs JSCoverage should work with conditionals",
+  "Coverage specs JSCoverage should work",
+  "device request prompt does not crash",
+  "Drag n' Drop should drag and drop",
+  "Drag n' Drop should drop using mouse",
+  "Drag n' Drop should drop",
+  "Emulate idle state changing idle state emulation causes change of the IdleDetector state",
+  "Emulation Page.emulate should support clicking",
+  "Emulation Page.emulate should work",
```

Comment:
Should both actually be included? they use user agent and viewport emulation.

Reply:
It also emulates isMobile/isLandscape/hasTouch so I don't expect these to work fully. But looks like the tests don't actually verify those but we might discover those are needed once other parts are in place.

Reply:
Right. So lets see.